designs/event-streams.md

```diff
@@ -236,11 +232,14 @@ class InputEventStream[I: SerializableShape, R](Protocol):
         await self.close()
 
 
-class OutputEventStream[O: DeserializableShape, R](Protocol):
+class OutputEventStream[OE: DeserializeableShape, O: DeserializeableShape]:
 
-    output_stream: AsyncEventReceiver[O]
-
-    response: R
+    output_stream: EventReceiver[OE]
+    response: O
```

Comment:
Why differing names for the same thing? `response` vs `output`, how about `initial_response` to be extra clear about what it is?

Reply:
I changed it to `output`. I want this to be the same in all of them and initial response doesn't make sense for input streams.